atom/common/native_mate_converters/v8_value_converter.cc

```diff
@@ -415,6 +415,11 @@ base::Value* V8ValueConverter::FromV8Object(
       child_v8 = v8::Null(isolate);
     }
 
+    // Ignore external values since calling CreationContext() on them can
+    // crash
+    if (child_v8->IsExternal())
```

Comment:
Can this check be done in `FromV8ValueImpl`? Otherwise crash may still happen when the external property is passed directly to `ipc.send`.